lms/templates/class_dashboard/metrics_click_handlers.js

```diff
@@ -0,0 +1,34 @@
+<%page args="section_data"/>
+
+$(function () {
+
+  // Click handler for x axis major ticks
+  $('.metrics-container').on('click', '.stacked-bar-graph-axis .tick.major', function () {
+
+    var stackedBarElement;
+    var xValue;
+    var moduleId;
```

Comment:
Could we ever encounter a case where the proper module is never found in the following `for` loop?
If so, `moduleId` would be `undefined` when attempting to build the URL.